src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2783,7 +2987,8 @@ because xbuild doesn't support framework reference assemblies.
 	<Delete Files="$(MonoAndroidIntermediate)__AndroidLibraryProjects__.zip" />
 	<Delete Files="$(MonoAndroidIntermediate)__AndroidNativeLibraries__.zip" />
 	<Delete Files="$(MonoAndroidIntermediate)stub_application_data.txt" />
-	<Delete Files="$(IntermediateOutputPath)_javac.stamp" />
+	<Delete Files="$(MonoAndroidIntermediate)_javac.stamp" />
+	<Delete Files="$(MonoAndroidIntermediate)compiled.flata" />
```

Comment:
This seems likewise wrong: the `_CompileResources` target in this PR creates `$(IntermediateOutputPath)compiled.flata`, not `$(MonoAndroidIntermediate)`.